phylogenetic/rules/prepare_sequences.smk

```diff
@@ -78,34 +48,34 @@ rule filter:
         min_length=config["filter"]["min_length"],
         exclude_where=lambda wildcard: config['filter']['exclude_where'][wildcard.build],
     shell:
-        """
+        r"""
         augur filter \
-            --sequences {input.sequences} \
-            --metadata {input.metadata} \
-            --metadata-id-columns {params.metadata_id_columns} \
-            --exclude {input.exclude} \
-            --sequence-index {input.index} \
-            --min-length {params.min_length} \
+            --sequences {input.sequences:q} \
+            --metadata {input.metadata:q} \
+            --metadata-id-columns {params.metadata_id_columns:q} \
+            --exclude {input.exclude:q} \
+            --sequence-index {input.index:q} \
+            --min-length {params.min_length:q} \
             {params.exclude_where} \
-            --output {output.sequences} \
-            --output-metadata {output.metadata}
+            --output-sequences {output.sequences:q} \
+            --output-metadata {output.metadata:q}
         """
 
 
 rule nextclade_before_mask:
     input:
         fasta="results/{build}/sequences.fasta",
-        reference=config["nextclade"]["reference_fasta"],
-        genemap=config["nextclade"]["genemap"],
+        reference=resolve_config_path(config["nextclade"]["reference_fasta"]),
+        genemap=resolve_config_path(config["nextclade"]["genemap"]),
     output:
         alignment="results/{build}/premask.fasta",
     shell:
-        """
+        r"""
         nextclade3 run \
-            --input-ref {input.reference} \
-            --input-annotation {input.genemap} \
-            --output-fasta {output.alignment} \
-            -- {input.fasta}
+            --input-ref {input.reference:q} \
+            --input-annotation {input.genemap:q} \
+            --output-fasta {output.alignment:q} \
+            -- {input.fasta:q}
```

Comment:
very nit, much pick: split this across two lines so it's more clearly intentional:

```suggestion
            -- \
            {input.fasta:q}
```